src/ssl.c

```diff
@@ -582,6 +582,56 @@ int wolfSSL_CTX_GenerateEchConfig(WOLFSSL_CTX* ctx, const char* publicName,
     return ret;
 }
 
+int wolfSSL_CTX_GetEchConfigsBase64(WOLFSSL_CTX* ctx, byte* output,
+    word32* outputLen) {
+    byte* rawBuffer;
+    int ret = 0;
+    word32 startLen;
+    word32 rawLen;
+
+    if (ctx == NULL || outputLen == NULL)
+        return BAD_FUNC_ARG;
+
+    /* if we don't have ech configs */
+    if (ctx->echConfigs == NULL) {
+        return WOLFSSL_FATAL_ERROR;
+    }
+
+    /* save off the buffer capacity */
+    startLen = *outputLen;
+
+    rawBuffer = (byte*)XMALLOC(*outputLen, ctx->heap, DYNAMIC_TYPE_TMP_BUFFER);
+
+    /* get the configs */
+    ret = GetEchConfigsEx(ctx->echConfigs, rawBuffer, outputLen);
```

Comment:
Do we not already have a version of this API `GetEchConfigsEx` the customer can call and base 64 encode?